CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Changed
+- Unified `DelayMs` into `DelayUs` with `u32` as only primitive for clarity.
```

Comment:
Hmm, do you think something along the lines of "Removed `DelayMs` in favor of `DelayUs` with `u32` as type for clarity." would be clearer?

Reply:
done

Reply:
Thanks!